axelrod/strategies/averagecopier.py

```diff
@@ -19,8 +19,7 @@ class AverageCopier(Player):
         'manipulates_state': False
     }
 
-    @staticmethod
-    def strategy(opponent):
+    def strategy(self, opponent):
```

Comment:
As was implemented here the strategy method was in fact static and not using self. By passing self you do make it a non static method but there is no advantage to doing so as the method does not use self.